benchmarks/benchmark_serving.py

```diff
@@ -664,6 +738,8 @@ def main(args: argparse.Namespace):
     else:
         raise ValueError(f"Unknown dataset: {args.dataset_name}")
 
+    slos_dict = check_goodput_args(args)
```

Comment:
The name `slos_dict` is confusing in this script. Maybe call it `gootput_config_dict` or `slo_config_dict`. I'll prefer to have `goodput` in the name because the purpose of this config is only for calculating goodput.